addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -338,7 +339,40 @@ export class WebsitePreview extends Component {
         this.title.setParts({ action: currentTitle });
     }
 
-    _onPageLoaded() {
+    _onPageLoaded(ev) {
+        // FIX Chrome-only. If you have the backend in a language A but the
+        // website in English only, you can 1) modify a record's (event,
+        // product...) name in language A (say "New Name").
+        // 2) visit the page `/new-name-11` => the server will redirect you to
+        // the English page `/origin-11`, which is the only one existing.
+        // Chrome caches the redirection.
+        // 3) give the same name in English as in language A, try to visit
+        // => the server now wants to access `/new-name-11`
+        // => Chrome uses the cache to redirect `/new-name-11` to `/origin-11`,
+        // => the server tries to redirect to `/new-name-11` => start again 🔄.
+        // If we detect that behavior, we reload the iframe with a new query
+        // parameter, making the URL brand-new for Chrome.
+        if (navigator.userAgent.toLowerCase().includes("chrome") && !this.iframeReloaded) {
+            try {
+                this.iframe.el.contentWindow.location.href;
+            } catch (err) {
+                const msgCheck = `Failed to read a named property 'href' from 'Location': Blocked a frame with origin "${location.origin}" from accessing a cross-origin frame.`;
+                if (err.name === "SecurityError" && err.message === msgCheck) {
```

Comment:
This is not an acceptable solution, it's really too fragile and going to break the second chrome update its message.
Maybe it's even translated if your browser is not in English, which would be even worst.

Can't you just catch `err.name === "SecurityError"`? At worst, if we catch more than what we should, there shouldn't be any harm to reload the url with the new param?

You should also explain in the comment why a CORS error is raised while loading the URL from the same domain, it's confusing.

It's because Chrome is injecting in the iframe content its own broken page layout with the broken smiley
Surely worth explaining
![image](https://github.com/odoo/odoo/assets/30048408/9866bc43-56cd-4925-861d-212e90a0a5f2)

Reply:
> Maybe it's even translated if your browser is not in English, which would be even worst.

I thought about that and checked, it's not the case. But I get your point.

> Can't you just catch err.name === "SecurityError"? At worst, if we catch more than what we should, there shouldn't be any harm to reload the url with the new param?

Yes I suppose you're right. If there's another security error, it will happen again on reload and it won't get caught in the try/catch the 2nd time.

I'll change that and update the comment.